build-system/babel-plugins/babel-plugin-transform-jss/test/fixtures/transform-assertions/should-transform-jss-var/output.mjs

```diff
@@ -0,0 +1,23 @@
+var _classes = JSON.parse("{\"button\":\"button-0-2-1\",\"CSS\":\".button-0-2-1 {\\n  font-size: 12px;\\n}\"}");
```

Comment:
Can we minify this CSS at all?

Reply:
Yes, but I'd like to defer that

Reply:
I hope that eventually we can run our CSS optimizer/prefixer code on these stylesheets.